src/ensembl/src/content/app/App.tsx

```diff
@@ -57,30 +67,33 @@ export const App: FunctionComponent<AppProps> = (props: AppProps) => {
     };
   }, [props.match.path]);
 
-  const { url } = props.match;
+  return (
+    <Suspense fallback={<div>Loading...</div>}>
+      <Switch>
+        <Route path={`${url}/global-search`} component={GlobalSearch} />
+        <Route path={`${url}/species-selector`} component={SpeciesSelector} />
+        <Route
+          path={`${url}/browser/:species/:stableId/`}
+          component={Browser}
+        />
+        <Redirect
+          exact={true}
+          from={`${url}/browser`}
+          to={{
+            pathname: `${url}/browser/GRCh38_demo/ENSG00000139618`,
+            search: '?region=13:32271473-32437359'
+          }}
+        />
+      </Switch>
+    </Suspense>
+  );
+};
 
+export const App: FunctionComponent<AppProps> = (props: AppProps) => {
   return (
-    <Fragment>
-      <AppBar currentApp={props.currentApp} />
-      <Suspense fallback={<div>Loading...</div>}>
-        <Switch>
-          <Route path={`${url}/global-search`} component={GlobalSearch} />
-          <Route path={`${url}/species-selector`} component={SpeciesSelector} />
-          <Route
-            path={`${url}/browser/:species/:stableId/`}
-            component={Browser}
-          />
-          <Redirect
-            exact={true}
-            from={`${url}/browser`}
-            to={{
-              pathname: `${url}/browser/GRCh38_demo/ENSG00000139618`,
-              search: '?region=13:32271473-32437359'
-            }}
-          />
-        </Switch>
-      </Suspense>
-    </Fragment>
+    <AppShell>
```

Comment:
Is using `Shell` different from using `Wrapper`?

Reply:
As a word? I don't think so. Just a different metaphor.

The phrase "app shell" is used a lot to describe a nearly empty box into which the app is going to be rendered.

Reply:
I like `AppShell` in this context. Will fit right into the initial rendered component if we use SSR.